scripts/change-org.js

```diff
@@ -0,0 +1,67 @@
+const glob = require('globby');
```

Comment:
If you wanted to do more neckbeard and less hipster, you could do something like:

```sh
find . -name '*package.json' -print0 | xargs -0 perl -pi -e 's#github.com/dtothefp/build-boiler/#github.com/build-boiler/build-boiler/#g'
```

:neckbeard: :neckbeard: :neckbeard: :neckbeard: